crates/axevent-sys/build.rs

```diff
@@ -10,7 +10,7 @@ fn populated_bindings(dst: &path::PathBuf) {
         .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
         .layout_tests(false);
     for path in library.include_paths {
-        bindings = bindings.clang_args(&["-F", &path.to_str().unwrap()]);
+        bindings = bindings.clang_args(&["-F", (path.to_str().unwrap())]);
```

Comment:
Why is this changed?

Reply:
Clippy started complaining about it.

Sorry about the lack of explanation in the PR description; Because this project is set to create the default commit message by concatenating all commits messages I am not updating the PR description and motivations are more likely to be found in commits prefixed `s:` for _squash_.